src/preact/compat.js

```diff
@@ -1,27 +1,30 @@
-import * as compat from /*OK*/ 'preact/compat';
-
-/**
- * @param {function(T, {current: (R|null)}):PreactDef.Renderable} fn
- * @return {function(T):PreactDef.Renderable}
- * @template T, R
- */
-export function forwardRef(fn) {
-  return compat.forwardRef(fn);
-}
-
 /**
- * @param {PreactDef.VNode} vnode
- * @param {HTMLElement} container
- * @return {PreactDef.VNode}
+ * todo(kvchari): replace with official solution from preact
```

Comment:
Add an issue for this.

Reply:
https://github.com/ampproject/amphtml/issues/36948